client-memory.html

```diff
@@ -11,22 +11,39 @@
 <div class="page-navigation">
 <h2>Contents</h2>
 <p>
+<a href="#introduction">Introduction</a><br />
 <a href="#macosx">OMERO.insight on OSX</a><br />
 <a href="#windows">OMERO.insight on Windows</a><br />
 <a href="#linux">OMERO.insight on Linux/Unix</a><br />
 </p>
 </div><!-- End Page Navigation -->
 
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2><a class="anchor" id="introduction"></a>Introduction</h2>
 
 <div class="line-block">
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<p>By default, the OMERO clients allocate a certain amount of memory for themselves during the start-up process which should be sufficient for most uses. However, when dealing with very large images, you may encounter out of memory exceptions while trying to import or display these large images. In these situations, it is beneficial to increase the memory allocation to the clients as outlined below.</p>
+<p>By default, the OMERO desktop clients allocate a certain amount of memory for themselves during the start-up process which should be sufficient for most uses.</p>
+
+<p>However, when dealing with very large images, you may encounter out of memory exceptions while trying to import or display these large images. In these situations, it is beneficial to increase the memory allocation to the clients as outlined below.</p>
```

Comment:
You have large images twice in this sentence which is a bit repetitive; "However, you may encounter out of memory exceptions while trying to import or display very large images." would read better.